appium/tests/screenobjects/email.screen.ts

```diff
@@ -52,7 +52,7 @@ class EmailScreen extends BaseScreen {
   }
 
   checkEmailAddress = async (email: string) => {
-    const selector = `-ios class chain:**/XCUIElementTypeStaticText[\`label == "${email}"\`]`;
+    const selector = `~${email}`;
     await (await $(selector)).waitForDisplayed();
```

Comment:
What I meant is, an accessibility identifier should be added to the sender element on message screen in Swift. Then query here by that identifier.

Reply:
eg 

```
Error: element ("~e2e.enterprise.test@flowcrypt.com") still not displayed after 15000ms
    at async EmailScreen.checkEmailAddress (/Users/tom/git/flowcrypt-ios/appium/tests/screenobjects/email.screen.ts:56:5)
    at async EmailScreen.checkOpenedEmail (/Users/tom/git/flowcrypt-ios/appium/tests/screenobjects/email.screen.ts:71:5)
    at async UserContext.<anonymous> (/Users/tom/git/flowcrypt-ios/appium/tests/specs/composeEmail/SendEncryptedEmailAfterPassPhraseSessionEndedAndTrashIt.spec.ts:57:5)
```

When in fact I see it on screen. By using accessibility identifiers instead, we remove uncertainty.

Reply:
added senderEmail accessibilityId